internal/cmd/cmd_zpopmax.go

```
@@ -0,0 +1,85 @@
+	}
+
+	sortedSet, errInfo := sortedset.FromObject(obj)
+	if errInfo != nil {
+		return cmdResNil, errors.ErrWrongTypeOperation
+	}
+
+	res := sortedSet.PopMax(count)
+	return cmdResSlice(res), nil
+}
+
+func parseZPOPMAXArgs(c *Cmd) (string, int, error) {
```

golangci-lint (1.64.8)

unnamedResult: consider giving a name to these results

(gocritic)